lib/resources/datasets.js

```diff
@@ -9,30 +9,22 @@
 
 const sanitize = require('sanitize-filename');
 const { getOrNotFound } = require('../util/promise');
-const { Form } = require('../model/frames');
-const { ensureFormDef } = require('../util/db');
 const { streamEntityCsvs } = require('../data/entity');
 const { contentDisposition } = require('../util/http');
 
 module.exports = (service, endpoint) => {
   service.get('/projects/:id/datasets', endpoint(({ Projects, Datasets }, { auth, params, queryOptions }) =>
     Projects.getById(params.id, queryOptions)
       .then(getOrNotFound)
-      .then((project) => auth.canOrReject('project.read', project)) // TODO change the verb
+      .then((project) => auth.canOrReject('dataset.list', project))
       .then(() => Datasets.getAllByProjectId(params.id))));
 
-  service.get('/projects/:projectId/forms/:id/draft/dataset-diff', endpoint(({ Forms, Datasets }, { params, auth }) =>
-    Forms.getByProjectAndXmlFormId(params.projectId, params.id, false, Form.DraftVersion)
-      .then(getOrNotFound)
-      .then((form) => auth.canOrReject('form.update', form)) // what verb is required here? a new one
-      .then(ensureFormDef)
-      .then(() => Datasets.getDatasetDiff(params.id))));
-
   service.get('/projects/:projectId/datasets/:name/download', endpoint(({ Datasets, Entities, Projects }, { params, auth }, _, response) =>
-    Projects.getById(params.projectId) // TODO: different lookup/auth scheme for datasets
+    Projects.getById(params.projectId)
       .then(getOrNotFound)
-      .then((project) => auth.canOrReject('project.read', project))
+      .then((project) => auth.canOrReject('dataset.read', project))
```

Comment:
Are datasets actees? Should the actee here be the dataset rather than the project?

Reply:
Datasets aren't currently actees.